### DISCUSSION TITLE
Miss data

### DISCUSSION THREAD

**Discussion by khphilippe** [dr5hn/countries-states-cities-database#702] [General]
Hi. First of all thans for this api. Its so useful. i would like to use it in a proyect in argentina but when i use https://api.countrystatecity.in/v1/countries/AR/states/A/cities the list is empty. Please can we add this cities? whithout that i can't use this beautiful api

**Comment by dr5hn** [comment]
@khphilippe Thank you for bringing this to my attention. May I request a reliable source where I can obtain an accurate list of cities for this state?
